src/Console/Commands/stubs/addon/package.json.stub

```diff
@@ -1,19 +1,17 @@
 {
     "private": true,
     "scripts": {
-        "dev": "npm run development",
-        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "watch-poll": "npm run watch -- --watch-poll",
-        "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "prod": "npm run production",
-        "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
+        "dev": "mix",
+        "development": "mix",
+        "watch": "mix watch",
+        "watch-poll": "mix watch -- --watch-options-poll=1000",
+        "hot": "mix watch --hot",
+        "production": "mix --production"
     },
     "devDependencies": {
-        "cross-env": "^5.1",
-        "laravel-mix": "^5.0.9",
-        "vue": "^2.6.12",
-        "vue-loader": "^15.9.5",
-        "vue-template-compiler": "^2.6.11"
```

Comment:
I think if vue-templpate-compiler is removed, then the (install) function for npm or yarn needs to be run twice, which will install the vue-template-compiler, because of the webpack file's `mix.js('resources/js/addon.js', 'dist/js').vue();` Is it worth just including it out of the box? so that one could just install node_modules and run dev at minimum and have the Vue field type ready to go?

Reply:
It didn't need to be run twice. Seemed fine to me.